File: SLAManager/SLAManager/SLAManager/Model/SlaMetricViolationForecast.cs
using System;
using System.Collections.Generic;

namespace SLAManager.Model;

public partial class SlaMetricViolationForecast
{
    public int Id { get; set; }

    public int? IdSla { get; set; }

    public string? Violation { get; set; }

    public DateTime? Datetime { get; set; }
}

File: SlaManagerService/app/Model/SlaMetricViolationForecast.py
class SlaMetricViolationForecast:
    def __init__(self, IdSla=None, Violation=None,Datetime=None,Action=None,Code=None,Controller=None,Endpoint=None,Instance=None,Job=None,Method=None,DesiredValue=None,MisuredValue=None):
        self.IdSla = IdSla
        self.Violation = Violation
        self.DesiredValue = DesiredValue
        self.MisuredValue = MisuredValue 
        self.Datetime = Datetime
        self.Action = Action
        self.Code = Code
        self.Controller = Controller
        self.Endpoint = Endpoint
        self.Instance = Instance
        self.Job = Job
        self.Method = Method
